arch/arm/configs/sdm660-perf_defconfig

```diff
@@ -17,8 +17,6 @@ CONFIG_CGROUP_FREEZER=y
 CONFIG_CPUSETS=y
 CONFIG_CGROUP_CPUACCT=y
 CONFIG_CGROUP_SCHEDTUNE=y
-CONFIG_MEMCG=y
```

Comment:
should these be also applied to our defconfigs or are these specific to sdm660? according to their Kconfig entries they seem helpful actually...
pinging @kholk (sorry!)